L/libcups/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcups"
+version = v"2.4.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/openprinting/cups.git", "12ff481989924fc82ffdf1b8699753a486a33547")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/cups.git
```

Comment:
I thought we fixed this bug already
```suggestion
cd ${WORKSPACE}/srcdir/cups
```

Reply:
Looks like https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/1203 isn't in any release it